network-mapper/templates/visibility-dns-deployment.yaml

```diff
@@ -13,6 +13,10 @@ spec:
       labels:
         app: {{ .Release.Name }}-dns
     spec:
+      securityContext:
```

Comment:
if not in values.yaml then value is null, shouldn't we remove the field entirely in this case?